Doc/library/selectors.rst

```diff
@@ -63,6 +63,11 @@ constants below:
    +-----------------------+-----------------------------------------------+
    | :const:`EVENT_WRITE`  | Available for write                           |
    +-----------------------+-----------------------------------------------+
+   | :const:`EVENT_EXCEPT` | Available for urgent data                     |
```

Comment:
Please call it `EVENT_URGENT` then. `EVENT_EXCEPT` is confusing in Python context (do I get an exception?).